src/controllers/users.js

```diff
@@ -1,16 +1,33 @@
-const { User } = require('../models');
+const { UsersService } = require('../services');
+const usersService = new UsersService();
```

Comment:
Make this field private, same as here https://github.com/Sf-62-NodeJS/clothes-market-server/blob/2dae079aa956d0614ab4171f7b618dede5ac8d01/src/controllers/sizes.js#L4

Reply:
Fixed